fable-test-app/src/components/SubmitHoliday/StepOne.tsx

```diff
@@ -0,0 +1,232 @@
+import { forwardRef, useEffect } from 'react';
+import { Text } from '../../components';
+
+import {
+  GcdsButton,
+  GcdsDateInput,
+  GcdsFieldset,
+  GcdsInput,
+  GcdsRadioGroup,
+  GcdsStepper,
+  GcdsTextarea,
+  GcdsCheckbox,
+  GcdsDetails,
+  GcdsSelect,
+  GcdsFileUploader,
+  GcdsLink,
+  GcdsErrorSummary
+}
+from '@cdssnc/gcds-components-react';
+
+interface StepOneProps {
+  formdata: {
+    holidayname: string,
+    newholiday: string,
+    holidaydate: string,
+    learnofholiday: string,
+    holidaytype: {
+      federal: boolean,
+      national: boolean,
+      other: boolean
+    },
+    otherholiday: string,
+    province: string,
+    image: string[] | null,
+  };
+  handleInputChange: (e: any) => void;
+  focusHeading: boolean;
+};
+
+const StepOne = forwardRef<HTMLGcdsStepperElement, StepOneProps>(( props, heading) => {
+
+  const { formdata, handleInputChange, focusHeading } = props;
+
+  const newHolidayOptions = [
+    { "label": "Yes", "id": "radio1", "value": "yes", "checked": formdata.newholiday === "yes"},
+    { "label": "No", "id": "radio2", "value": "no", "checked": formdata.newholiday === "no"},
+    { "label": "Not sure", "id": "radio3", "value": "notsure", "checked": formdata.newholiday === "notsure"}
+  ];
+
+  const provinces: any = {
+    AB: "Alberta",
+    BC: "British Columbia",
+    MB: "Manitoba",
+    NB: "New Brunswick",
+    NL: "Newfoundland and Labrador",
+    NS: "Nova Scotia",
+    NT: "Northwest Territories",
+    NU: "Nunavut",
+    ON: "Ontario",
+    PE: "Prince Edward Island",
+    QC: "Québec",
+    SK: "Saskatchewan",
+    YT: "Yukon"
+  };
```

Comment:
Could we have a central location for this? In Melanie's other PR, I've asked if we can have a file called `utils/constants.js`, I think this would be great to be put in there too, I see all over the app repeatedly defined in different ways.